observability/opentelemetry-metrics-backend/src/main/scala/sttp/client4/opentelemetry/OpenTelemetryMetricsBackend.scala

```diff
@@ -1,23 +1,23 @@
 package sttp.client4.opentelemetry
 
+import io.opentelemetry.api.OpenTelemetry
 import io.opentelemetry.api.common.Attributes
 import io.opentelemetry.api.metrics.{DoubleHistogram, LongCounter, LongUpDownCounter, Meter}
-import io.opentelemetry.api.OpenTelemetry
-import sttp.client4.{wrappers, _}
 import sttp.client4.listener.{ListenerBackend, RequestListener}
 import sttp.client4.wrappers.FollowRedirectsBackend
+import sttp.client4._
 
 import java.time.Clock
 import java.util.concurrent.ConcurrentHashMap
 
 object OpenTelemetryMetricsBackend {
-  val DefaultLatencyHistogramName = "sttp_request_latency"
-  val DefaultRequestsInProgressCounterName = "sttp_requests_in_progress"
-  val DefaultSuccessCounterName = "sttp_requests_success_count"
-  val DefaultErrorCounterName = "sttp_requests_error_count"
-  val DefaultFailureCounterName = "sttp_requests_failure_count"
-  val DefaultRequestSizeHistogramName = "sttp_request_size_bytes"
-  val DefaultResponseSizeHistogramName = "sttp_response_size_bytes"
+  val DefaultLatencyHistogramName = "sttp.request.duration"
```

Comment:
can we add some comments here which point to the conventions used? both for OT and Prometheus. I suppose using the `sttp` prefix is fine, there are no standard labels to use?

Reply:
I will add a comment 
these one https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-client describes the standard labels and metrics for HTTP client, I just decide to change `http.client` prefix to `sttp` beside that labels are the same

Reply:
BTW should we use Long histogram for request/response sizes instead of current Double histogram?

Reply:
> BTW should we use Long histogram for request/response sizes instead of current Double histogram?

I have no idea :) How to others do it?

Reply:
Ok so I will change to `http.client prefix`
Not sure as to the histogram in convention it is describe as experimental metric.
I just realized that we do not expect floating point bytes in these histogram so Long one should be sufficient